client/client_test.go

```diff
@@ -3274,6 +3274,85 @@ func TestDeleteRepo(t *testing.T) {
 	requireRepoHasExpectedKeys(t, repo, rootKeyID, true)
 }
 
+// TestDeleteRemoteRepo tests that local and remote repo data is deleted from the client library call
+func TestDeleteRemoteRepo(t *testing.T) {
+	gun := "docker.com/notary"
+
+	ts := fullTestServer(t)
+	defer ts.Close()
+
+	// Create and publish a repo to delete
+	repo, rootKeyID := initializeRepo(t, data.ECDSAKey, gun, ts.URL, false)
+	defer os.RemoveAll(repo.baseDir)
+
+	require.NoError(t, repo.Publish())
+
+	// Create another repo to ensure it stays intact
+	livingGun := "stayingAlive"
```

Comment:
![Image of turtle dancing to staying alive](http://s2.quickmeme.com/img/7b/7b3580d674b71dbcb3a8e06a996a75b6a0b08db731d96ade298b71f5461c8b70.jpg)